cmd/nerdctl/images.go

```diff
@@ -144,6 +148,30 @@ func imagesAction(cmd *cobra.Command, args []string) error {
 	return printImages(ctx, cmd, client, imageList)
 }
 
+func filterByReference(imageList []images.Image, filters []string) ([]images.Image, error) {
+	var filteredImageList []images.Image
+	logrus.Debug(filters)
+	for _, image := range imageList {
+		logrus.Debug(image.Name)
+		var matches int
+		for _, f := range filters {
+			var err error
+			matchFound, err := regexp.MatchString(f, image.Name)
```

Comment:
Looks like it is expected to support wildcard, not regexp?
https://docs.docker.com/engine/reference/commandline/images/#filter-images-by-reference

```
docker images --filter=reference='busy*:*libc'
```

Reply:
We should also make sure that `--filter=reference='busy*:*libc'` matches `docker.io/library/busy*:*libc`